src/main.py

```diff
@@ -1,20 +1,72 @@
+import os
+import toml
+import argparse
+
 from generate_csv import generate_csv
 from csv_to_cda import csv_to_cda
 from calculate_dependencies import calculate_dependencies
-from src.sent_to_server_and_print_error import send_xml_file
 
-if __name__ == '__main__':
-    csv_path = '../resources/data.csv'
-    excel_path = '../resources/CDAVariables.xlsx'
-    xslt_file = '../resources/EmergencyNote.xslt'
 
-    # First step: Generate
-    generate_csv(excel_path, csv_path, 10)
+def set_environment_variables(config):
+    """
+    Set environment variables based on the provided configuration dictionary.
+
+    Args:
+        config (dict): Configuration dictionary containing paths for environment variables.
+
+    Returns:
+        None
+    """
+    env_vars = [
+        ('CSV_PATH', 'cda_paths.csv_path'),
+        ('EXCEL_PATH', 'cda_paths.excel_path'),
+        ('XSLT_FILE', 'cda_paths.xslt_file'),
+        ('OUTPUT_DIR', 'cda_paths.output_dir'),
+        ('CITIES_CSV', 'csv_paths.cities_csv'),
+        ('DIAGNOSES_CSV', 'csv_paths.diagnoses_csv')
+    ]
+
+    for env_var, config_key in env_vars:
+        section, key = config_key.split('.')
+        os.environ[env_var] = config[section][key]
+
+
+def parse_command_line():
+    global parser, args, rows
+    parser = argparse.ArgumentParser(description='Process Excel to CDA.')
+    parser.add_argument('--rows', type=int, required=True, help='Number of rows to generate in the CSV file')
+    args = parser.parse_args()
+    rows = args.rows
+
+
+def main():
```

Comment:
When running the script I get following errors:

```
Traceback (most recent call last):
File "/home/akomi/PycharmProjects/cda-test-data-generator/src/main.py", line 72, in <module>
main()
File "/home/akomi/PycharmProjects/cda-test-data-generator/src/main.py", line 65, in main
csv_to_cda(csv_path, xslt_file)
File "/home/akomi/PycharmProjects/cda-test-data-generator/src/csv_to_cda.py", line 115, in csv_to_cda
output_dir = os.environ['output_dir']
File "/usr/lib/python3.10/os.py", line 680, in __getitem__
raise KeyError(key) from None
KeyError: 'output_dir'
Error in sys.excepthook:
Traceback (most recent call last):
File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 72, in apport_excepthook
from apport.fileutils import likely_packaged, get_recent_crashes
File "/usr/lib/python3/dist-packages/apport/__init__.py", line 5, in <module>
from apport.report import Report
File "/usr/lib/python3/dist-packages/apport/report.py", line 32, in <module>
import apport.fileutils
File "/usr/lib/python3/dist-packages/apport/fileutils.py", line 28, in <module>
from apport.packaging_impl import impl as packaging
File "/usr/lib/python3/dist-packages/apport/packaging_impl.py", line 23, in <module>
import apt
ModuleNotFoundError: No module named 'apt'

Original exception was:
Traceback (most recent call last):
File "/home/akomi/PycharmProjects/cda-test-data-generator/src/main.py", line 72, in <module>
main()
File "/home/akomi/PycharmProjects/cda-test-data-generator/src/main.py", line 65, in main
csv_to_cda(csv_path, xslt_file)
File "/home/akomi/PycharmProjects/cda-test-data-generator/src/csv_to_cda.py", line 115, in csv_to_cda
output_dir = os.environ['output_dir']
File "/usr/lib/python3.10/os.py", line 680, in __getitem__
raise KeyError(key) from None
KeyError: 'output_dir'
```

Reply:
I don't really know how can I reproduce this behaviour